.github/bin/dist-github-release.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/ash
```

Comment:
```suggestion
#!/bin/sh
```

or, did you intend [amquist shell](https://en.wikipedia.org/wiki/Almquist_shell)? (If so, is there any special reason?)

Reply:
Yes, it's Amquist shell by intention, because it's closest to Bash and there's no bash in this Docker image. 😭

Reply:
So are these scripts okay to run with `/bin/sh`?

- https://github.com/planetarium/libplanet/pull/235/files#diff-4a84523e74f96c4e221c2e75647af2ceR1
- https://github.com/planetarium/libplanet/pull/235/files#diff-906da86a1ded732ffc59f7d5e075a846R1

Reply:
These are not intention, maybe. I will make these to use ash too.

Reply:
Ah, these were intention! These two scripts are run on `mcr.microsoft.com/dotnet/core/sdk:2.2` image, which have neither bash nor ash… it has only POSIX sh. 😇